Apps/W1/Shopify/app/src/Products/Codeunits/ShpfyCreateItemAsVariant.Codeunit.al

```diff
@@ -0,0 +1,110 @@
+namespace Microsoft.Integration.Shopify;
+
+using Microsoft.Inventory.Item;
+
+codeunit 30343 "Shpfy Create Item As Variant"
```

Comment:
I think this could use a little tests. We are already well covered in codeunit 139601 "Shpfy Create Product Test". That would be a good place to start.

Reply:
@tinestaric fyi